db/migrate/20180814202747_add_recurring_logic_to_sync_plan.rb

```diff
@@ -1,13 +1,14 @@
 class AddRecurringLogicToSyncPlan < ActiveRecord::Migration[5.1]
   def change
-    add_column :katello_sync_plans, :foreman_tasks_recurring_logic_id, :integer
-    add_column :katello_sync_plans, :cron_expression, :string
-    add_foreign_key :katello_sync_plans, :foreman_tasks_recurring_logics, :name => "katello_sync_plan_foreman_tasks_recurring_logic_fk", :column => "foreman_tasks_recurring_logic_id"
-    Katello::SyncPlan.find_each do |sync_plan|
-      User.as_anonymous_admin do
-        sync_plan.associate_recurring_logic
-        sync_plan.save!
-      end
-    end
+    # add_column :katello_sync_plans, :foreman_tasks_recurring_logic_id, :integer
+    # add_column :katello_sync_plans, :cron_expression, :string
+    # add_foreign_key :katello_sync_plans, :foreman_tasks_recurring_logics, :name => "katello_sync_plan_foreman_tasks_recurring_logic_fk", :column => "foreman_tasks_recurring_logic_id"
+    # Katello::SyncPlan.find_each do |sync_plan|
+    #   User.as_anonymous_admin do
+    #     sync_plan.associate_recurring_logic
+    #     sync_plan.save!
+    #   end
+    # end
+    # Moved to migration CreateRepositoryRoot to resolve upgrade conflicts
```

Comment:
I would just leave the "moved" comment and remove the others (preferred since people can look it up via git history if needed), or put the "moved" comment up top to be more visible